draft-irtf-nwcrg-coding-and-congestion-04.xml

```diff
@@ -89,19 +89,19 @@
         <!-- ######################################################-->
 
     <abstract>
-	    <t>FEC coding is a reliability mechanism that is distinct and separate from the loss detection of congestion controls. Using FEC coding can help deal with transfer tail losses or with networks having non-congestion losses. However, FEC coding mechanisms should not hide congestion signals. This memo offers a discussion of how FEC coding and congestion control can coexist. Another objective is to encourage the research community to also consider congestion control aspects when proposing and comparing FEC coding solutions in communication systems.</t>
+	    <t>Forward Erasure Correction (FEC) is a reliability mechanism that is distinct and separate from the retransmission logic in reliable transfer protocols such as TCP. Using FEC coding can help deal with transfer tail losses or with networks having non-congestion losses. However, FEC coding mechanisms should not hide congestion signals. This memo offers a discussion of how FEC coding and congestion control can coexist. Another objective is to encourage the research community to also consider congestion control aspects when proposing and comparing FEC coding solutions in communication systems.</t>
```

Comment:
This is an important change, because it's the first sentence of the abstract. I didn't like that this says that FEC is a reliability mechanism that is distinct from the loss detection ...   because loss detection and reliability are two different things already. Anyway, we say "FEC coding mechanisms should not hide congestion signals" so it's also redundant.

Reply:
That looks better to me. Thanks.